### PR TITLE
LibGUI: Make sure combobox list windows can't be moved

### PR DESCRIPTION
**Pull request by gunnarbeutner**
https://user-images.githubusercontent.com/388571/197605103-accd1171-71c9-4ec2-849b-5a095aad9ccb.mp4

### CODE REVIEW

File: Userland/Libraries/LibGUI/ComboBox.cpp [hunk 1]
```diff
@@ -113,6 +113,7 @@ ComboBox::ComboBox()
     };
 
     m_list_window = add<Window>(window());
+    m_list_window->set_window_type(GUI::WindowType::Tooltip);
```
Comment (linusg) [inline Userland/Libraries/LibGUI/ComboBox.cpp:116]: This seems extremely brittle

Comment (gunnarbeutner) [reply Userland/Libraries/LibGUI/ComboBox.cpp:116]: I figured I'd do the same thing like `AutocompleteProvider`. Do you think adding a dedicated window type for this (and probably also `AutocompleteProvider` plus some other window in the `Locator` class) makes sense? How about `Popup` or `TemporaryPopup`?

Comment (linusg) [reply Userland/Libraries/LibGUI/ComboBox.cpp:116]: if it's architectured as a window, I think having a distinct window type would make a lot of sense, yep :)

**Review by linusg — APPROVED** [review]
Better!

**Pull request merged**